README.md

```diff
@@ -159,15 +159,28 @@ Important Notes:
 
 ## Error States
 
+### Error 400: Bad Request, no primary key
+If a WAL record for a table that does not have a primary key is passed through `realtime.apply_rls`, an error is returned
+
+Ex:
+```sql
+(
+    null,                            -- wal
+    null,                            -- is_rls_enabled
+    [...],                              -- users,
```

Comment:
@olirice replace `users` with `subscription_ids`. There are a few other places in README where `users` is still referenced.

Reply:
thanks, resolved